src/test/java/uk/ac/ic/wlgitbridge/application/WLGitBridgeIntegrationTest.java

```diff
@@ -159,10 +162,36 @@
     private MockSnapshotServer server;
     private GitBridgeApp wlgb;
     private File dir;
+    // set env-var "TEST_DB_MODE", either "postgres" or "sqlite"
+    private static String databaseMode;
+    private Map<String, String> postgresConfig = new HashMap<String, String>() {{
+      put("url", "jdbc:postgresql://postgres_v2/gitbridge_test");
+      put("username", "sharelatex");
+      put("password", "sharelatex");
+    }};
+
+    @BeforeClass
+    public static void before() throws Exception {
+      databaseMode = System.getenv("TEST_DB_MODE");
+      Log.info("Test database mode: " + databaseMode);
+    }
 
     @Before
     public void setUp() throws Exception {
-        dir = folder.newFolder();
+      dir = folder.newFolder();
+      if ("postgres".equals(databaseMode)) {
+        try {
+          PostgresDBStore dbStore = new PostgresDBStore(
+            postgresConfig.get("url"),
+            postgresConfig.get("username"),
+            postgresConfig.get("password")
+          );
+          dbStore.dangerousResetDatabaseDontDoIt();
```

Comment:
Oh, I see why this method exists now. Is there an easy way to localise this method to the test file, so we can keep it out of the main implementation? If there's no easy way to do it then don't worry about it, but it would be better to have dangerous code localised to the tests.